src/envs/cover.py

```diff
@@ -686,7 +689,8 @@ def _Pick_terminal(self, s: State, m: Dict, o: Sequence[Object],
                        p: Array) -> bool:
         # Pick is done when we're holding the desired object.
         del m, p  # unused
-        return self._Holding_holds(s, o)
+        block = o[0]
```

Comment:
Write this as `block, = o` instead to automatically allow the code to crash if the length of `o` is not what we would expect (i.e., 1).